packages/financial-templates-lib/src/price-feed/CreatePriceFeed.js

```diff
@@ -566,6 +566,17 @@ async function createUniswapPriceFeedForFinancialContract(
 
   const userConfig = config || {};
 
+  // Check if there is an override for the getTime method in the price feed config. Specifically, we can replace the
+  // get time method with the current block time.
+  if (userConfig.getTimeOverride) {
+    if (userConfig.getTimeOverride.useBlockTime) {
+      getTime = async () =>
+        web3.eth.getBlock("latest").then(block => {
+          return block.timestamp;
+        });
+    }
+  }
```

Comment:
```suggestion
  if (userConfig.getTimeOverride?.useBlockTime) {
    getTime = () => web3.eth.getBlock("latest").then(block => block.timestamp);
  }
```

Or

```js
  if (userConfig.getTimeOverride?.useBlockTime) {
    getTime = async () => (await web3.eth.getBlock("latest")).timestamp;
  }
```

Reply:
nice! I updated it as well as the previous instance of this override logic.